src/terrain/util/config.clj

```diff
@@ -504,6 +504,18 @@
        (cfg/env-setting "USER_PREFERENCES_PORT")
        (prefs-base-url)))))
 
+(cc/defprop-optstr search-base-url
+  "The hostname of the search service"
+  [props config-valid configs]
+  "terrain.search.base-url" "http://search:60000")
+
+(def search-base
+  (memoize
+   (fn []
+     (if (System/getenv "SEARCH_PORT")
+       (cfg/env-setting "SEARCH_PORT")
```

Comment:
Should we name this `SEARCH_URL` or something? The `else` is returning `search-base-url`...

Reply:
hm, I think we don't use the environment-based configs anyway; I was copying the others which have the same weirdness. I think I'll just remove this and use `search-base-url` directly